tests/test_verilog/test_from_file.py

```diff
@@ -24,7 +24,7 @@ def check_rxmod(RXMOD):
     check_port(RXMOD, "data", m.Array, "output")
     check_port(RXMOD, "valid", m.Bit, "output")
 
-    m.compile("build/test_rxmod", RXMOD, output="verilog")
+    m.compile("build/test_rxmod", RXMOD)
```

Comment:
This tests compiling a top level wrapped verilog module using the coreir backend